docs/10X_STAR_Solo_alignment.md

```diff
@@ -31,7 +31,8 @@ They can also be obtained from the following sources:
  - `r2_max_len`: Set read2 to a fixed length. Useful for single cell experiments in which the number of cycles was improperly configured during sequencing. Recommend `91` for 10X v3 chemistry
 
 ### STAR Solo
- - `outSAMattrRGline`: Set if outputting bam, with TABS SEPARATING THE TAGS, format is: ID:sample_name LB:aliquot_id PL:platform SM:BSID for example ID:7316-242	LB:750189	PL:ILLUMINA	SM:BS_W72364MN
+ - `outSAMattrRGline`: Set if outputting bam, with TABS SEPARATING THE TAGS, format is: ID:sample_name LB:aliquot_id PL:platform SM:BSID for example `ID:HFWFVDMXX.L001	LB:750189	PL:ILLUMINA	SM:BS_W72364MN`. If more than one fastq pair in input, it is recommended to have one read group per pair, separated by space-comma-space (` , `)
```

Comment:
I think we should make this a list input since BixOps seems to prefer that approach.

Reply:
Test with no BAM: https://cavatica.sbgenomics.com/u/d3b-bixu/kf-dev-single-cell-rna/tasks/5effafe4-6db8-4961-9268-41b585ed05ec/
Test with BAM: https://cavatica.sbgenomics.com/u/d3b-bixu/kf-dev-single-cell-rna/tasks/8b14fe1d-2d9b-4545-8953-e8d3b4b6f202/